test/gurka/gurka.cc

```diff
@@ -239,7 +239,8 @@ inline void build_pbf(const nodelayout& node_locations,
                       const nodes& nodes,
                       const relations& relations,
                       const std::string& filename,
-                      const uint64_t initial_osm_id) {
+                      const uint64_t initial_osm_id,
+                      const bool strict) {
```

Comment:
i added a new option to not complain when you make osm that isnt perfect (ie simulate real world mappers osm input data)